site/docs/uk/guide/deployment-types.md

```diff
@@ -226,7 +226,27 @@ app.use(webhookCallback(bot, "oak"));
 
 > Зауважте, що ви не повинні викликати `bot.start()` при використанні вебхуків.
 
-Обовʼязково прочитайте [чудовий путівник Марвіна про все, що стосується вебхуків](https://core.telegram.org/bots/webhooks), написаний командою Telegram, якщо ви плануєте запустити свого бота на вебхуках у VPS.
+Тепер ваш додаток слухає запити на вебхук від Telegram.
```

Comment:
Idk, this sentence in the original text looks weird...

Reply:
```suggestion
Тепер ваш додаток прослуховує запити на вебхук від Telegram.
```